osutil/disks/disks.go

```diff
@@ -145,9 +145,16 @@ func parseUdevProperties(r io.Reader) (map[string]string, error) {
 	return m, scanner.Err()
 }
 
-// DiskFromName finds a matching Disk using the specified name, such as vda, or
-// mmcblk0, etc.
+// DiskFromDeviceName finds a matching Disk using the specified name, such as
+// vda, or mmcblk0, etc.
 func DiskFromDeviceName(deviceName string) (Disk, error) {
+	return diskFromDeviceName(deviceName)
+}
+
+// diskFromDeviceName is exposed for mocking from other tests via
+// MockDeviceNameDisksToPartitionMapping, but has the same problems as
```

Comment:
The comment says "but has the same problems as..." - what are these problems?

Reply:
these problems don't actually exist since we don't use diskFromDeviceName in multiple places like we do with diskFromMountPoint and diskFromMountPointImpl, so this comment should really be simplified to just say this is a var for mocking